src/test/java/io/vertx/ext/dropwizard/MetricsTest.java

```diff
@@ -353,14 +347,14 @@ public void testHttpMetricsOnClose() throws Exception {
     client.close();
     server.close(ar -> {
       assertTrue(ar.succeeded());
-      testComplete();
+      vertx.setTimer(100, tid -> testComplete());
```

Comment:
maybe instead of adding a delay, doing just:

```java
vertx.runOnContext(v -> testComplete());
```

Is faster and does the same. the `runOnContext` can also be seen as node.js `process.nextTick` a simple way to execute something on the next event loop cycle, this allows the pending operations to terminate.